### PR TITLE
Authentication set-up complete for now(will implement handling nicknames later)

### PR DESCRIPTION
**Pull request by hilakey**
This is the step I am trying to implement
**Hide the comments form by default. Modify your script.js file to fetch the login status from the servlet. If the user is logged in, unhide the form. If the user is not logged in, display a login link.**

I made some comments in my script.js and LoginDataServlet.java files about some of the ideas I had to do it. I just wanted to see if what I'm thinking of doing can be done or if there's another way around it.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/LoginDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    UserService userService = UserServiceFactory.getUserService();
+    response.getWriter().println("<h2>USER LOGIN STATUS!</h2>");
+    if (userService.isUserLoggedIn()) {
+      String urlToRedirectToAfterUserLogsOut = "/status";
+      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+
+      response.getWriter().println("<p>Hello user!</p>");
+      response.getWriter().println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+
+      //need a way to communicate with JS that this is true, to unhide form(boolean)
```
Comment (hilakey) [inline portfolio/src/main/java/com/google/sps/servlets/LoginDataServlet.java]: Here's where I'm trying to see if a JS function can be called from here.
Or if not how to get the response to be evaluated in the JS function

File: portfolio/src/main/java/Comments.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package usercomments;
+
+public class Comments{
```
Comment (gagomez) [inline portfolio/src/main/java/Comments.java]: nit: Singular form (Comment) since this class represents a single comment.

**Review by gagomez — APPROVED** [review]
Nice job!

**Review by jiaxinz-g — APPROVED** [review]

File: portfolio/src/main/java/Comments.java (hunk 2 above)
Comment (hilakey) [reply portfolio/src/main/java/Comments.java]: Done!

**Pull request merged**